crates/ruff_python_formatter/src/statement/suite.rs

```diff
@@ -459,6 +471,51 @@ impl Format<PyFormatContext<'_>> for SuiteChildStatement<'_> {
     }
 }
 
+pub(crate) struct FormatClauseBody<'a> {
+    body: &'a Suite,
+    kind: SuiteKind,
+    trailing_comments: &'a [SourceComment],
+}
+
+impl<'a> FormatClauseBody<'a> {}
```

Comment:
We'll need a method to set the `kind` so that `class` and `functions` can set the right suite kind.